README.md

```diff
@@ -40,58 +40,60 @@ This collection provides a list of educational resources for Software Engineers.
 
 ## The Software Development section consists of :
 
-### 1. Web Development - Here are just 5 from our huge range :
+### 1. Web Development - Here are just 6 from our huge range :
+
 - [Learn Enough to Be Dangerous](https://www.learnenough.com/) - Free online coding tutorials on JavaScript, Ruby, Rails, CSS and more.
 - [FreeCodeCamp](https://www.freecodecamp.org/) - Coding tutorials and challenges.
 - [Thimble](https://foundation.mozilla.org/en/artifacts/thimble/) - Free online code editor, web server, web browser & developer tools.
 - [NodeSchool](https://nodeschool.io/) - Open source workshops that teach web software skills.
 - [The Odin Project](https://www.theodinproject.com/) - A full free open source coding curriculum.
+- [W3Schools](https://www.w3schools.com/) - A fully fledged website to learn web development through tutorials, references, and examples.
```

Comment:
Please can you remove the w3 resource, as we already have that in the file named web development. Thank you @kenhorlador